src/main/java/fact/utils/ElementwiseMultiplyDoubleArray.java

```diff
@@ -0,0 +1,46 @@
+package fact.utils;
+
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+import fact.features.singlePulse.timeSeriesExtraction.ElementWise;
```

Comment:
Maybe wrong place, maybe not. You started using a method outside its first intended scope ;) So it is this code point where it got evident for the first time.